.github/workflows/qa.yml

```diff
@@ -1,6 +1,7 @@
 # Runs quality assurance checks
 name: "qa"
 on:
+  pull_request:
```

Comment:
We don't want to run this workflow on every pull request because it is slow and it will hence slow down development. We usually only want to run QA tests when we are inside a release branch.